jet_tracking/jet_control.py

```diff
@@ -303,21 +361,32 @@ def jet_move_inline(injector, camera, params):
         # move the ROI to keep looking at the jet
         min_x = ROIx + (params.jet_x.get() / params.pxsize.get())
         camera.ROI.min_xyz.min_x.put(min_x)
-    # if params.state == [some state]
-    #     [use [x] for jet tracking]
-    # else if params.state == [some other state]:
-    #     [use [y] for jet tracking]
-    # else if params.state == [some other state]:
-    #     [revert to manual injector controls]
-    # etc...
-
-    # if jet is clear in image:
-    #     if jet_x != beam_x:
-    #         move injector.coarseX
-    #         walk_to_pixel(detector, motor, target) ??
-    # else if nozzle is clear in image:
-    #     if nozzleX != beam_x:
-    #         move injector.coarseX
-    # else:
-    #     if injector.coarseX.get() != beam_x:
-    #         move injector.coarseX
+
+
+def jet_scan(injector, cspad, gas_det, params):
```

Comment:
This will go away, right?

Reply:
Yep

Reply:
My understanding is that once jet_tracking detector a sub-par liquid scattering ring, it will start a scan of the injector motor to find the maximum, move there and go on with just monitoring the system. Is jet_scan that or something else?
There might also be code that constantly moves the motor to always stay at the peak - this code was to remain, but the scientists were not big fans of that method.